### PR TITLE
feat: add function to update account settings and tests

### PR DESCRIPTION
**Pull request by ramniel**
- [x] add tests to get and update account settings.
- [x] add function to update account settings.
- [x] add doubles for updating account settings.

### CODE REVIEW

**Comment by cbernat** [comment]
Why not integrate the get into the component before adding more loose method into an unused api?

File: src/services/doppler-contact-policy-api-client.test.ts [hunk 1]
```diff
@@ -1,15 +1,121 @@
...
+    const dopplerContactPolicyApiClient = createHttpDopplerContactPolicyApiClient({ request });
+
+    // Act
+    const result = await dopplerContactPolicyApiClient.getAccountSettings(emailAccount);
+
+    // Assert
+    expect(request).toBeCalledTimes(1);
+    expect(result).not.toBe(undefined);
+    expect(result.success).toBe(true);
+    expect(result.value.excludedSubscribersLists).not.toBe(undefined);
```
Comment (cbernat) [inline src/services/doppler-contact-policy-api-client.test.ts:63]: How about validating this other values as well?
  ```
  active: true,
  emailsAmountByInterval: 20,
  intervalInDays: 7,
  ```

**Comment by ramniel** [comment]
> Why not integrate the get into the component before adding more loose method into an unused api?

The idea is to define the service with the tests in this PR and then the integration in the next one.

**Comment by cbernat** [comment]
👍

**Pull request merged**